src/main/java/ui/components/pickers/LabelPickerState.java

```diff
@@ -0,0 +1,271 @@
+package ui.components.pickers;
+
+import backend.resource.TurboLabel;
+import util.Utility;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class LabelPickerState {
+    List<String> addedLabels;
+    List<String> removedLabels;
+    Set<String> initialLabels;
```

Comment:
Move this to the variable to the top of the list. This order is more intuitive: initial, added, removed, matched, suggested